src/keymap-manager.coffee

```diff
@@ -598,6 +598,48 @@ class KeymapManager
       currentTarget = currentTarget.parentNode ? window
     return
 
+  # Public: Simulate keypresses on a DOM node.
+  #
+  # This can be useful in tests, when you want to simulate keyboard input that is
+  # not bound to a command in the CommandRegistry.
+  #
+  # * `target` The DOM node at which to start bubbling the key events
+  # * `key` {String} indicating the key to press.
+  # * `modifierKeys` {Object} indicating which modifier keys are pressed. Possible
+  #    keys are `ctrl`, `shift`, `alt`, `meta`
```

Comment:
I think `meta` should probably be `cmd` for consistency. I know that doesn't quite match the DOM event spec but all the keystroke strings elsewhere in the API use `cmd`.